bin/copyProjectDataToPrm.sh

```diff
@@ -57,10 +57,12 @@ Usage:
 
 Options:
 
-	-h   Show this help.
-	-g   Group.
-	-n   Dry-run: Do not perform actual sync, but only list changes instead.
-	-l   Log level.
+	-h	Show this help.
+	-g	Group.
+	-p	[pipeline]
+		from which pipeline is the data coming from (NGS_Demultiplexing, GAP)
```

Comment:
from ... from is a bit strange. How about:
`Pipeline that produced the project data that needs to be transferred to prm.`

Reply:
fixed